prog/weave-kube/weave-monitoring-svc.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+   name: weave-net-monitoring
+   namespace: kube-system
+   annotations:
+     prometheus.io.scrape: "true"
+spec:
+   selector:
+      name: weave-net
```

Comment:
FWIW, the [kubelint](https://github.com/weaveworks/service-conf/blob/master/k8s/kubelint) script will bork on this, insisting that the `name` selector match the service name.

This probably isn't an issue for your repo, but I'd recommend keeping consistent with `service-conf` unless you have a compelling reason to do otherwise.